test/helpers/compute_resources_helper_test.rb

```diff
@@ -20,7 +20,7 @@ def initialize(context)
       compute.stubs(:datacenters).raises(Foreman::FingerprintException, 'Wrong fingerprint')
       instance.controller.action_name = 'test_connection'
       instance.list_datacenters(compute)
-      compute.errors.messages[:pubkey_hash].first.must_include 'Wrong fingerprint'
+      _(compute.errors.messages[:pubkey_hash].first).must_include 'Wrong fingerprint'
```

Comment:
I wonder, might these conflict with our usage of `_()` for string extraction and translations somehow? I don't think we scan the test folder for strings to translate, but https://github.com/theforeman/foreman/blob/develop/lib/foreman/gettext/all_domains.rb#L5 defines the same function and https://github.com/theforeman/foreman/blob/develop/config/initializers/1_fast_gettext.rb#L33 includes it during initialization

Reply:
The minitest cops are limited to the `test` folder by their default configuration.

If this leads to problems, we'll have a lot of fun when we want to go to minitest 6. :smile: